code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -349,7 +349,8 @@
 	icon = 'icons/mob/critter.dmi'
 	icon_state = "viscerator_attack"
 	icon_living = "viscerator_attack"
-	pass_flags = PASSTABLE
+	pass_flags = PASSTABLE | PASSMOB
```

Comment:
Is the idea that for this to work, swarming enemies should have `PASSMOB` set?

Reply:
Well, for mobs, yes.

For everything else? Not really.

It's just mobs have this weird aspect where they can only occupy the same turf if one happens to be lying down, initially (or have passmob); otherwise they bump into each other and swap spots.